src/plugins/Wallet/database/Wallet.db.ts

```diff
@@ -9,9 +9,9 @@ export const createWalletDBAccess = createDBAccess(() => {
     return openDB<WalletDB>('maskbook-plugin-wallet', 1, {
         upgrade(db, oldVersion, newVersion, tx) {
             function v0_v1() {
-                db.createObjectStore('RedPacket', { keyPath: path<keyof RedPacketRecord>('id') })
-                db.createObjectStore('ERC20Token', { keyPath: path<keyof ERC20TokenRecord>('address') })
-                db.createObjectStore('Wallet', { keyPath: path<keyof WalletRecord>('address') })
+                db.createObjectStore('RedPacket', { keyPath: path<keyof RedPacketRecordInDatabase>('id') })
```

Comment:
Does it be able to migrate old data (raw bigint) or support both of them?

Reply:
need old data sample, verifying the issue.